webpack.config.ts

```diff
@@ -146,6 +146,7 @@ export default (env: {minimize?: boolean} = {}, argv: {mode?: string}): webpack.
                             "hotkey_hints",
                         ],
                         preventIndent: true,
+                        inlineRequires: "((\\.\\./)+(generated/emoji/)?images/)",
```

Comment:
Should this have a comment? Also would it be simpler to do a `|` for `generated/emoji/images|images/`?

Reply:
I'll add a comment.

I don't find the alternation necessarily simpler, but I'm ambivalent about it, so I'll adjust.